server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -324,6 +340,11 @@ export function configureWebSocketServices(
 
                     // eslint-disable-next-line max-len
                     logger.error(`Disconnecting socket on connection error: ${safeStringify(error, undefined, 2)}`, { messageMetaData });
+                    Lumberjack.log(
```

Comment:
Will it be useful to add a metric for connectDocument? It is a critical part of the flow and it will not be as noisy as the lambdas, so it might be a good candidate?

Reply:
For this metric, are we thinking
1. Start when connection is established
2. Fail when disconnecting on error (right around where this comment is)
3. Succeed when disconnection volunarily?